### ISSUE TITLE
MK8 Question

### ISSUE THREAD

**Issue by meluvalli** [dimok789/loadiine_gx2#138]
I know that MK8 only works with Smash Bros mod, but was wondering if this will ever work with Mii Maker or is it a limitation on system itself.   I know it's a v64 issue, but I am not sure what that means :)

**Comment by vinny** [comment]
**v64** is the update version. if your game isn't updated, you can run in both methods, SSB and MiiMaker.

**Comment by meluvalli** [comment]
Yes.  I'm aware of this.  I am wondering if Mii Maker will ever work with it, or is it a limitation on the nintendo Mii maker app.   I am wondering because I don't own Smash Bros...  So, for me to be able to play backup version, I would have to purchase Smash Bros just for this...  If there's a chance that loadiine will support it eventually, I probably will hold off, but if it's a limitation on Mii Maker, then I'll probably buy it.

**Comment by Twilight** [comment]
so... just not use the update?

**Comment by robo502** [comment]
use this backup to play MK8 is the MK8 without updates run good in loadiine you just need to find the meta folder but you can play without the meta folder too https://docs.google.com/uc?id=0B-j9anQrMvjCbHFPOU9GdENfdlU&export=downloa

**Comment by janssenr** [comment]
Does MK8 with v64 only work with SSB? I tried KaraokeU. But that doesn't work on my WiiU 5.5.1E
